backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/repos/AdministratorsRepository.kt

```diff
@@ -34,7 +34,7 @@ object AdministratorsRepository {
         return resultRow?.let {
             val user = AdministratorEntity.wrapRow(it)
             val passwordHash = user.passwordHash
-            if (passwordHash !== null && PasswordCrypto.verifyPassword(password, passwordHash)) {
+            if (passwordHash != null && PasswordCrypto.verifyPassword(password, passwordHash)) {
```

Comment:
Why those changes?

Reply:
In Kotlin, structural equality (what we usually want) is the `==` operator (whereas `===` is the ref equality). (https://kotlinlang.org/docs/equality.html)
That's quite annoying because for JS we usually want to use the `===` operator. (I guess JS is to blame here)